src/sql/azdata.proposed.d.ts

```diff
@@ -432,6 +432,54 @@ declare module 'azdata' {
 		azurePortalEndpoint?: string;
 	}
 
+
+	export namespace diagnostics {
+		/**
+		 * Represents a diagnostics provider of accounts.
+		 */
+		export interface ErrorDiagnosticsProviderMetadata {
+			/**
+			 * The name of the provider (ex. a connection provider) that a diagnostics provider will handle errors for.
```

Comment:
Clarify this is the ID (since some providers have displayName as well)

```suggestion
			 * The id of the provider (ex. a connection provider) that a diagnostics provider will handle errors for.
```